asaps/models.py

```diff
@@ -66,20 +66,23 @@ def download_json(rec_obj):
     f = open(file_name + '.json', 'w')
     json.dump(rec_obj, f)
     f.close()
+    return f.name
 
 
-def createcsv(csv_data, file_name):
+def create_csv(csv_data, file_name):
     """Create CSV file from list of dicts.
 
     Example: {'uri': rec_obj.uri,
     'old_value': old_value , 'new_value': new_value}.
     """
     date = datetime.datetime.now().strftime('%Y-%m-%d %H.%M.%S')
     header = list(csv_data[0].keys())
-    f = csv.DictWriter(open(file_name + date + '.csv', 'w'), fieldnames=header)
+    full_file_name = file_name + date + '.csv'
```

Comment:
Using a relative filename is fine if its use is limited to local scope. In this case, since you're returning it, I would definitely use an absolute path to avoid any confusion elsewhere:
```python
full_file_name = os.path.abspath(f"{file_name}{date}.csv")
```